packages/openapi-sampler/src/samplers/StringSampler.ts

```diff
@@ -107,6 +140,33 @@ export class StringSampler implements Sampler {
     return value;
   }
 
+  private assertLength(min?: number, max?: number) {
+    const pairs = [
+      { key: 'minLength', value: min },
+      { key: 'maxLength', value: max }
+    ];
+
+    const boundariesStr = pairs
+      .filter((p) => !this.checkBoundary(p.value))
+      .map((p) => `${p.key}=${p.value}`)
+      .join(', ');
+
+    if (boundariesStr) {
+      throw new Error(
+        `Sample string cannot be generated by boundaries: ${boundariesStr}`
+      );
+    }
+  }
+
+  private checkBoundary(boundary?: number) {
+    return (
+      boundary === undefined ||
+      (typeof boundary === 'number' &&
+        boundary >= 0 &&
+        boundary <= this.MAX_PATTERN_SAMPLE_LENGTH)
+    );
```

Comment:
The check is redundant; do not validate the schema

```suggestion
    return (
      boundary === undefined ||
      boundary <= this.MAX_PATTERN_SAMPLE_LENGTH
    );
```